src/home.py

```diff
@@ -68,4 +68,4 @@ def home():
     )
 
     st.header("Read more on Medium")
-    get_blogs()
+    # get_blogs()
```

Comment:
Remove comment

Reply:
I think the `get_blogs()` is broken, should be commented out